app.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+function StoreSalesGenerator(name, minCust, maxCust, avgSale) {
+  this.storeName = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgSale = avgSale;
+  this.numCookies = [];
+  this.totalSales = 0;
+  this.hour = 6;
+}
+
+var newText = document.createTextNode('text');
+var tableRow, tableHeader, tableData;
+var position = document.getElementById('sales-estimates');
+
+var storeHour = function(){
+  tableRow = document.createElement ('tr');
+  position.appendChild(tableRow);
+  tableData = document.createElement ('td');
+  tableRow.appendChild(tableData);
+  tableData = document.createElement ('td');
+  for (var hour = 6; hour < 20; hour++ ) {
+    tableHeader = document.createElement ('th');
+    if (hour < 12 ) {
+      newText = document.createTextNode(hour + ' am ');
+      tableHeader.appendChild(newText);
+      tableRow.appendChild(tableHeader);
+    } else if (hour == 12) {
+      newText = document.createTextNode('12pm ');
+      tableHeader.appendChild(newText);
+      tableRow.appendChild(tableHeader);
+    } else {
+      newText = document.createTextNode((hour-12) + 'pm ');
+      tableHeader.appendChild(newText);
+      tableRow.appendChild(tableHeader);
+    }
+  }
+  tableHeader = document.createElement ('th');
+  newText = document.createTextNode('Daily Location Totals');
+  tableHeader.appendChild(newText);
+  tableRow.appendChild(tableHeader);
+};
+
+StoreSalesGenerator.prototype.estimatedCookies = function(){
```

Comment:
Split this function into two functions, one that generates your hourly sales and one that adds the sales data to the table.  A function should do one thing only.